### DISCUSSION TITLE
part-ii/2018-131/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#217] [Question comments]
Part II 2018 Representation Theory: Paper 2, Section II, I 

https://questions.tripos.org/part-ii/2018-131/

**Comment by Kiolt** [comment]
<details>
<summary>Part (i) and (ii).</summary>

Induce the restricted character, it decomposes as $\chi+\lambda\chi$. Use part (a).

</details>

<details>
<summary>Part (c).</summary>

Again use part (a). For the first case take $C_3$ and any representation. For the second case take $G=S_3$ and $K=C_2$ and the irreducible 2-dim representation. For the last case take $G=A_4$ and $K=V$ and the irreducible 3-dim representation.

</details>
